lrauv_system_tests/test/vehicle/CMakeLists.txt

```
@@ -34,6 +34,14 @@ target_link_libraries(test_dvl
     lrauv_gazebo_plugins::lrauv_gazebo_messages)
 gtest_discover_tests(test_dvl)
 
+add_executable(test_dvl_acoustic_comms test_dvl_acoustic_comms.cc)
+target_link_libraries(test_dvl_acoustic_comms
+  PUBLIC gtest_main
+  PRIVATE
+    ${PROJECT_NAME}_support
+    lrauv_gazebo_plugins::dvl_velocity_tracking)
```

@adityapande-1995 should be: 
```suggestion
    lrauv_gazebo_plugins::lrauv_gazebo_messages)
```

---

Done